src/riak_ql_parser.yrl

```diff
@@ -931,6 +950,8 @@ get_func_type(FuncName) when FuncName =:= 'AVG'    orelse
                              FuncName =:= 'STDDEV_SAMP' orelse
                              FuncName =:= 'STDDEV_POP' ->
     window_aggregate_fn;
+get_func_type('TIME') ->
+    sql_select_fn;
```

Comment:
More functions will use this AST tag, at the moment `now()` will use it. The function name in the AST describes what the function is.

I had a look into re-factoring the function handling code to unify the ASTs but it is no small task, and I didn't want to conflict with your own function work.